openmc/cell.py

```diff
@@ -197,7 +196,14 @@ def rotation(self):
 
     @property
     def rotation_matrix(self):
-        return self._rotation_matrix
+        if self.rotation is not None:
```

Comment:
Yeah, I think this will help speed things up, based on my experience doing this sort of thing in OpenCG anyway.